canonical-web-and-design/canonical.com
# Update the links based on the data source

## Done
Update the links based on the data source

## QA
- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8002/careers
- Click through the links and ensure they don't 404.


## Issue / Card
Fixes https://github.com/canonical-web-and-design/canonical.com/issues/278
